components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -4697,4 +4701,28 @@ public static String resolveExternalConsentPageUrl(String tenantDomain) throws I
 
         return externalConsentPageUrl;
     }
+
+    /**
+     * Check whether the application should be FAPI conformant.
+     *
+     * @param clientId       Client ID of the application.
+     * @return Whether the application should be FAPI conformant.
+     * @throws OAuthClientAuthnException
+     */
+    public static String isFapiConformantApp(String clientId) throws OAuthClientAuthnException {
```

Comment:
Shall we change the return type to boolean?

Reply:
Fixed in 0ec55c25fc65cdef29f0cebfb16d87d104ec0ec3